src/fixHashes.ts

```diff
@@ -0,0 +1,38 @@
+import { getExecOutput } from "@actions/exec";
+
+export interface Mismatch {
+  readonly derivation: string;
+  readonly replacement: string;
+}
+
+export interface Fix {
+  readonly line: number;
+  readonly found: string;
+  readonly mismatches: readonly Mismatch[];
+}
+
+export interface FileFix {
+  readonly file: string;
+  readonly fixes: readonly Fix[];
+}
+
+export interface FixHashesOutputV1 {
+  readonly version: "v1";
+  readonly files: readonly FileFix[];
+}
+
+export async function getFixHashes(): Promise<FixHashesOutputV1> {
+  const output = await getExecOutput("determinate-nixd", [
+    "fix",
+    "hashes",
+    "--json",
+  ]);
```

Comment:
```suggestion
  const output = await getExecOutput("determinate-nixd", [
    "fix",
    "hashes",
    "--json",
  ], {
    silent: true,
  });
```

without silent: true the log gets littered with ugly output:

![CleanShot 2025-04-14 at 11 50 47](https://github.com/user-attachments/assets/275a9d16-0264-4336-9658-17a1507b7098)